shared/ui/components/form/inputs/LocalIdentitiesAutoComplete.js

```diff
@@ -14,16 +14,16 @@ import {
 const withKey = item => ({ key: item.address, ...item })
 const sortAlphAsc = (a, b) => a.name.localeCompare(b.name)
 
-const LocalIdentitiesAutoComplete = React.memo(
```

Comment:
Why remove memo here?

Reply:
The linter complained that the `displayName` property for this component was not there. I don't think `memo` actually gets us anything here; see [When to useMemo and useCallback](https://kentcdodds.com/blog/usememo-and-usecallback).